src/lib/url.js

```diff
@@ -0,0 +1,64 @@
+const queryParams = ['include', 'fields', 'sort'];
+
+export const parseInclude = include => {
+  return include ? include.split(',') : [];
+};
+
+export const parseFields = query => {
+  let fields = {};
+
+  if (query) {
```

Comment:
When would this be false?

Reply:
Ah never I guess- it is always an instance of `URLSearchParams` - I'll remove this.